### ISSUE TITLE
Program fails to open

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#4]
Program fails to open when attempted:
after clicking on mixx beta icon on the dektop I am prompted with a message asking my if I would like to participate in the Mixx'x development community feedback:
I am presented with the yes-No-Privacy Policy buttons

Upon clicking yes the wondow closes and mixx continues to run (i presume).
Mixx does not actually appear, instead I am left with my desktop just as it was when I had opened it. However, if I ctrl+alt+delete and click the processes tab, mixxx.exe is running at about 0 CPU usage and 47-49,000 KB of memory usage. after about...3-5 seconds, mixx.exe dissappears from the processes tab and never appears again.

IF however I decline to participate in the mixxx development community, I am presented with the windows error message informing me that mixxx.ese has encountered a problem and needs to close window. and I am offered to send error report (to microsoft I assume) and "don't send"

after this I decided to run Mixxx 1.5 on my computer. only to find that it often crashes when trying to load an mp3 into either of the tracks.

for what its worth: I am running
AMD athlon XP 2500+ (1.83 ghz)
1.25 GB RAM
nVidia GeForce 6600 GT 128MB
on a Soyo KT880 Dragon 2 v.2 Motherboard (discontinued to my knowledge)

for what its worth, I am also attaching the error report that I was promted to send to microsoft because I wouldnt trust them to spend more than a minute with it before throwing it out.

**Comment by Holzhaus** [comment]
(by poolstud634)

**Comment by Holzhaus** [comment]
(by gamegod)
John, this might be a CMetrics bug. Any hints on troubleshooting?

**Comment by Holzhaus** [comment]
(by gamegod)
poolstud634: This bug is top priority, and I was wondering if you could please try a few things to help us track it down:

1) Go into your Mixxx directory (probably C:\Program Files\Mixxx) and see if there's a "mixxx.log" file. If so, please upload it here so we can take a look at it.

2) After you check for the log (and have uploaded it if it exists), please download and run this new installer that has been built without cmetrics support:

http://downloads.mixxx.org/mixxx-1.6.0-svn/mixxx-1.6.0-svn06212008-win.exe

Thanks

**Comment by Holzhaus** [comment]
(by poolstud634)
hey
sorry for making you guys wait so long. I dont usually sign into this daily.
I did look in my Mixx directory and did not find a mixxx.log file...but i found a mixx.txt file...(i assume its the same thing...it seems to log a bunch of crap from mixx that I have no idea what it means)
anyways..I can upload that.....I can also say that I no longer get the request to perticipate in the development community...but...mixx will still not open at all!

Upon installing 1.6 beta 3 my computer thinks for a moment and then I am presented with a window that says:



(window heading)
Microsoft Visual C++ Runtime Library


Runtime Error!

Program Files\Mixx\mixxx.exe

this application has requested the Runtime to terminate it in an unusual way.
please contact applications support team for more information.



my only option is to hit ok and then i am presented with the regular windows crash windos that allows me to end task and such.

I am including the log file that windows is giving me,. And i am also including the log (txt) file that you requested. I would love to include some sort of hardware profile for you guys so you can see what exactlly the system is that im runing on but I do not know how to create such a log. I can also tell you that I have installed the Mixx 1.6 beta 3 on my laptop also with a completley different setup and processor even and it still did the same thing. I can also say that I have reformatted my computer since my first attempt to clean my hard drive and still the same message.

**Comment by Holzhaus** [comment]
(by poolstud634)
Here is the log file you requested as well

**Comment by Holzhaus** [comment]
(by rryan)
Hi poolstud634,

  Thanks very much for the logfile. Could you also post your mixxx.cfg? It's located in c:\Documents and Settings\YourUserName\mixxx.cfg.

Thanks again,
RJ

**Comment by Holzhaus** [comment]
(by rryan)
Poolstud634, 

  Just a quick question -- is this your first time running Mixxx? Have you installed a previous version before? Did you uninstall the previous version before installing this one?   

Thanks,
RJ

**Comment by Holzhaus** [comment]
(by poolstud634)
I have used mixxx since 1.5 and have had success running the beta with mild success since 1.6 b1
1.6b1 worked for the most part, but since 1.6 b2 the program has either crashed regularily upon loading mp3's, or crashing at start rendering it un-usable.

I am attaching the cfg file you requested.
I have in the past just installed newer copies of the beta over the old one, as well as completley uninstalling mixx and reinstalling. I have also gone as far as reformatting and still not being able to run mixxx.

**Comment by Holzhaus** [comment]
(by gamegod)
Poolstud634,

I've uploaded another build, this time compiled with more debugging output on:
http://downloads.mixxx.org/mixxx-1.6.0-svn/mixxx-1.6.0-svn07022008-win.exe
(upload should be done in about 15 mins)

If you could please test this and upload the mixxx.log file, it would be greatly appreciated. This build will probably also crash for you, but the debugging output is much more verbose and should help us pinpoint exactly where it's crashing.

Thanks,
Albert

**Comment by Holzhaus** [comment]
(by gamegod)
Poolstud634,

Our latest release is available here:
http://downloads.mixxx.org/mixxx-1.6.0-beta4/mixxx-1.6.0-beta4-win.exe

Please test this version as soon as you can and let us know if you still experience this problem. Your feedback is crucial to our success, as we have no way of reproducing this bug. We urgently need you to test the above package so we can allocate more resources to fixing other bugs.

Thank you,
Albert

**Comment by Holzhaus** [comment]
(by pegasus-renegadetech)
I am also seeing a similar problem on Vista Enterprise. This is a clean install of Vista (though running under a virtual PC) and Mixxx 1.6.0-beta4. If I answer Yes, Mixxx is in the processes list, taking up ~5MB of RAM but nothing comes up on the screen and it doesn't create the config files in the user profile. If I kill it, then re-run it and answer No, it comes up fine and works fine.
Mixxx.log is attached after killing it the first time.

**Comment by Holzhaus** [comment]
(by pegasus-renegadetech)
Tested with a non-cmetrics install and this problem goes away.

**Comment by Holzhaus** [comment]
(by poolstud634)
I apologize it has taken me so long to reply
mycomputer recieved a virus and i have since then had to reformat.
I can also note that after downlaoding the newest release of Mixxx....the problem still stands. I have installed a fresh copy of XP and mixxx was the first program i tried out. I can also say im running SP3 now, but I can install the newest release and send you what you need.

**Comment by Holzhaus** [comment]
(by poolstud634)
ok...I just installed the first release you provided for me:
http://downloads.mixxx.org/mixxx-1.6.0-svn/mixxx-1.6.0-svn07022008-win.exe


after installing that and attempting to run, I did not recieve the error in teh same fashion as before. Before it would come up with the windows has terminated the program, such n such like in my first post. Now, I get a different error. I am providing a screencap of the error window. I also looked for a .cfg file and a log file after installing and running this version. I could not find either.

after sending this I will install the other version in your later post:
http://downloads.mixxx.org/mixxx-1.6.0-beta4/mixxx-1.6.0-beta4-win.exe

I will send any .cfg and log files it produces
